### PR TITLE
Script to run STARlight anchored productions

### PR DESCRIPTION
**Pull request by mbroz84**
For STARlight production would be very handy if we can pass arguments to the job script in grid_submit.sh, so I did this small adjustment.

### CODE REVIEW

File: GRID/utils/grid_submit.sh [hunk 1]
```diff
@@ -575,7 +576,7 @@ export PATH=$PATH:$PWD
 # ----------- EXECUTE ACTUAL JOB  ------------------------------------ 
 # source the actual job script from the work dir
 chmod +x ./alien_jobscript.sh
-./alien_jobscript.sh
+./alien_jobscript.sh ${SCRIPTARGS}
```
Comment (sawenzel) [inline GRID/utils/grid_submit.sh:579]: This is unfortunately not as easy as this. One would, in addition, need to pass around the arguments in the JDL specification. Honestly, I would avoid doing this change for now. You can easily write a script which creates jobfiles containing the right arguments before submitting.

Comment (mbroz84) [reply GRID/utils/grid_submit.sh]: Ok, I reverted the changes in the grid_submit script.

**Comment by mbroz84** [comment]
+async-label async-2023-pbpb-apass4

**Comment by chiarazampolli** [comment]
Ported to async-v1-01-18, label removed.

**Pull request merged**